src/entities/supportRequest.ts

```diff
@@ -42,8 +52,8 @@ export class SupportRequest extends BaseEntity {
   @Column({ nullable: true })
   primaryLanguage: string;
 
-  @Column('simple-json')
-  extraData: { problemDescription: string };
+  @Column({ type: 'jsonb', nullable: true })
+  extraData: SupportRequestExtraData;
```

Comment:
Since this is nullable for the database, we should represent that in the type

```suggestion
  @Column({ type: 'jsonb', nullable: true })
  extraData: SupportRequestExtraData | null;
```